roles/ec2-app/node-metadata.md

```diff
@@ -58,6 +58,34 @@ Examples
         ruleset:
           - proto: 'tcp'
             ports: '443'
+      scaling_policies:
+        - name: scale_up
+          adjustment_type: "Percent"
+          min_adjustment_step: +1
+          scaling_adjustment: 100
+          cooldown: 180
+          metric_alarms:
+            - metric: "CPUUtilization"
+              comparison: ">="
+              threshold: 40
```

Comment:
@brettminnie not 45 here?

Reply:
oh - it get overridden in the local env pillar?

Reply:
That's example data, part of why I said don't merge, need to add the dynamodb stuff there too